src/formatDate.js

```diff
@@ -9,6 +9,57 @@
  */
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const divider = fromFormat[3];
```

Comment:
The variable name 'divider' could be more descriptive, consider renaming it to 'currentSeparator' to indicate its purpose more clearly.